ontology/yaml/resources/states/states.yaml

```diff
@@ -90,3 +90,14 @@ SOFTSTART: "Mode of operation of rectifier where the input power supply is gradu
 NORMAL: "Function, state or flow which is an ideal or expected condition."
 REVERSED: "Function, state or flow which is opposed to the expected or normal condition."
 
+# For elevator 
+GROUND: "Elevator floor status"
+FIRST: "Elevator floor status"
+SECOND: "Elevator floor status"
+THIRD: "Elevator floor status"
+FOURTH: "Elevator floor status"
+FIFTH: "Elevator floor status"
+SIXTH: "Elevator floor status"
+
+PARKING: "Elevator returning to default floor"
```

Comment:
Change to something like: "Elevator status indicator for when the elevator is returning to the default floor"